src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    public static void convertToLowerCase(Collection<String> strings)//This can use for any other Collection data type: eg.Set
```

Comment:
Good job on pulling this method out and making it more general-purpose, but you missed the header comment :(